7-extract_plier_pathways.R

```diff
@@ -65,6 +65,14 @@ norm.train.files <- file.path(
 )
 
 # define output files
+pathways_filename <- file.path(
+  plot.data.dir,
+  str_c(file_identifier, "_PLIER_pathways",
+        ifelse(permute, # distinguish between permuted/not permuted output files
+               ".permuted.tsv",
+               ".tsv"))
+)
+
```

Comment:
I do understand we're worried about file size, but it seems possible that the design decision not to keep the full models around could become one we regret, not unlike the situation we find ourselves in now!